src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar--day-start--wed calendar--month-length--28">
```

Comment:
Check, please, how it works when you change `calendar--day-start--wed` to `calendar--day-start--month` and other days of weeks in dev tools.

Also, check to the loop wich makes modifieres `--28` to set number of days. We haven't month langth one, two, three ... days.

Reply:
Found the problem you were referring to (weekday numbering started from 1, while it should start from 0) and fixed it.
Also prevented the possibility of building a month with less than 28 days.